sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/DerivationToolExt.kt

```diff
@@ -1,16 +1,15 @@
 package cash.z.ecc.android.sdk.internal
 
 import cash.z.ecc.android.sdk.internal.model.JniUnifiedSpendingKey
-import cash.z.ecc.android.sdk.model.Account
 import cash.z.ecc.android.sdk.model.UnifiedFullViewingKey
 import cash.z.ecc.android.sdk.model.UnifiedSpendingKey
 import cash.z.ecc.android.sdk.model.ZcashNetwork
 
 fun Derivation.deriveUnifiedAddress(
     seed: ByteArray,
     network: ZcashNetwork,
-    account: Account
-): String = deriveUnifiedAddress(seed, network.id, account.value)
+    accountIndex: Int
```

Comment:
This is different to Swift, where `Zip32AccountIndex` which wraps a `UInt32` is used.

Reply:
We could add this change. Let's use the same wrapper, but with the only difference in the inner parameter type as `Long`, and check the value for its scope as we do, e.g., in the case of `BlockHeight` [parameters](https://github.com/Electric-Coin-Company/zcash-android-wallet-sdk/blob/6f9ab065df6a5c277ad6174759c02c3ca2fd0c01/backend-lib/src/main/java/cash/z/ecc/android/sdk/internal/model/JniRewindResult.kt#L19). We can't use unsigned numbers across SDK as they are not supported by Java, which we need for many APIs used in protobug files.

Reply:
Done